### PR TITLE
Add LICENSE & VERIFICATION chocolatey requirements to nuspec

### LINKED ISSUE #137: chocolatey install ServiceBusExplorer doesnt work
this is what it says:

```
Chocolatey v0.10.7
Installing the following packages:
ServiceBusExplorer
By installing you accept licenses for the packages.
ServiceBusExplorer not installed. The package was not found with the source(s) listed.
 If you specified a particular version and are receiving this message, it is possible that the package name exists but the version does not.
 Version: ""
 Source(s): "https://chocolatey.org/api/v2/"

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - ServiceBusExplorer - ServiceBusExplorer not installed. The package was not found with the source(s) listed.
 If you specified a particular version and are receiving this message, it is possible that the package name exists but the version does not.
 Version: ""
 Source(s): "https://chocolatey.org/api/v2/"
```

### PR DESCRIPTION
**Pull request by Nate-Wilkins**
Fixes #137  - Adds appropriate verification files.

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.0.102-PullRequest0142.3.build.283 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/build/4.0.102-PullRequest0142.3.build.283) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/0ee2be8d08 by @Nate-Wilkins)

**Comment by SeanFeldman** [comment]
:x: [Build ServiceBusExplorer 4.0.102-PullRequest0142.4.build.284 failed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/build/4.0.102-PullRequest0142.4.build.284) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/d9349a62d6 by @Nate-Wilkins)

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.0.102-PullRequest0142.5.build.285 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/build/4.0.102-PullRequest0142.5.build.285) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/8bf8d1c6ce by @Nate-Wilkins)

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.0.102-PullRequest0142.6.build.286 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/build/4.0.102-PullRequest0142.6.build.286) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/8bf8d1c6ce by @Nate-Wilkins)

File: appveyor.yml [hunk 1]
```diff
@@ -49,6 +49,23 @@ build:
 #---------------------------------#
 
 after_build:
+  - ps: >-
+      # Remove ALL xml files.
```
Comment (tomkerkhove) [inline appveyor.yml]: Maybe it's good if we move all these steps into dedicated PowerShell scripts, if AppVeyor supports this?
ie. `Remove-XmlFiles`, `Reset-ChocolateyVerificationFile`, `Create-ChocolateyVerificationFile`

What do you think @SeanFeldman?

File: VERIFICATION.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (tomkerkhove) [inline VERIFICATION.txt]: I think we'll have to follow their format - [Example](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/automatic/x2go/legal/VERIFICATION.txt)

We could create the list and "tokenize" this example. Not sure if they enforce this or not though

**Comment by SeanFeldman** [comment]
I'm still thinking a single file reference is a better approach, but if
this is working, let's get over with it. It was the longest PR ever 😃

**Comment by tomkerkhove** [comment]
Merging it in, we can improve later on, if required.

**Pull request merged**